postcss.config.js

```diff
@@ -0,0 +1,3 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Why the blank postcss.config.js file? What is this being used for?

Reply:
Talked with @JakeLaCombe offline and he mentioned there is an [issue logged](https://github.com/cerner/terra-site/issues/35) for this.